packages/crayons-styles/scripts/convertTokens.js

```diff
@@ -0,0 +1,81 @@
+const fs = require('fs');
+const path = require('path');
+
+const tokenOutDir = './dist/design-tokens/';
```

Comment:
does this script after creating dist folder by rollup ?.

Reply:
No, The scripts runs before the rollup. All the design token sets are combined into a single based on the deisgn system and resulting file is created in dist folder. Which will be picked up by rollup and converted into js files for dynamic import.